tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/service/http/sample/MutualSSLTestCase.java

```diff
@@ -64,7 +64,7 @@ public void testMutualSSL() throws Exception {
         clientLeecher.waitForText(20000);
     }
 
-    @AfterClass (enabled = false)
+    @AfterClass ()
```

Comment:
Shall we remove the empty parenthesis here as well?

Reply:
Removed.